be2-scala/src/main/scala/ch/epfl/pop/storage/DbActor.scala

```diff
@@ -205,7 +205,7 @@ final case class DbActor(
 
   // generates the key of the RollCallData to store in the database
   private def generateRollCallDataKey(laoId: Hash): String = {
-    s"${ROLL_CALL_DATA_PREFIX}${laoId.toString}"
```

Comment:
You can also remove completely this defined `String`, found in the file `Channel.scala` I think.